ivy/utils/backend/handler.py

```diff
@@ -358,7 +358,8 @@ def set_backend(backend: str, dynamic: bool = False):
         global ivy_original_dict
         if not backend_stack:
             ivy_original_dict = ivy.__dict__.copy()
-
+        # copy current cython_wrappers_mode to the original dict
+        ivy_original_dict["cython_wrappers_mode"] = ivy.cython_wrappers_mode
```

Comment:
I think the value of the cython wrappers mode would be preserved across backends, similar to the other global modes, so we can probably revert the changes to this file

Reply:
I think the code will continue to work as expected even if we remove this line as we're using a stack now